index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/73a06aba35.js" crossorigin="anonymous"></script>
+    <title>Portfolio2</title>
+  </head>
+  <body>
+
+
+    <div class="container">
+      <header>
+        <a id="logo" href="index.html">Welcome</a>
+        <nav>
+          <i id="hamburger" class="fa-solid fa-bars"></i>
+        </nav>
+      </header>
+        
+
+      <section class="main-section">
+        <p class="primary-text">
+          <span>Hey there.I'm<br>Hammed<br></span> 
+          <span>I'm a Software Developer</span>
+        </p>
+
+        <p class="secondary-text">
+            I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        
+
+        <ul class="socials">
+          <li><a href="https://github.com/mansa-susa/" aria-label="visit my github profile"><i class="fab fa-github"></i></a></li>
+          <li><a href="https://www.linkedin.com/in/faruq-hammed-931862169/" aria-label="visit my linkedin profile"><i class="fa-brands fa-linkedin-in"></i></a></li>
+          <li><a href="#" aria-label="visit my angel list profile"><i class="fa-brands fa-angellist"></i></a></li>
+          <li><a href="https://twitter.com/mansa_susa" aria-label="visit my twitter profile"><i class="fa-brands fa-twitter"></i></a></li>
+          <li><a href="#" aria-label="visit my medium profile"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+        </ul>
```

Comment:
- [ ] Great job adding the social icons as a list, kindly try to increase the size of them and center them to match the Figma design.

![image](https://user-images.githubusercontent.com/89282221/185762594-93b20912-b1d9-472b-886d-7b23a0117930.png)